_docs/business-payments.md

```diff
@@ -128,7 +128,7 @@ CODE_EXAMPLE_END
 
     // Contact Details
     "phone_country": "US",
-    "phone_number": "5555551234",
+    "phone_number": "+15555551234", // E.164 international format
```

Comment:
did we use the code generator to generate these lines? Can't remember if they do comments properly hence my question

Reply:
No Zsolt I don't think we used the generator the first time for these files. Also I have tried to run the `build` to see if anything would be conflicting and it seems that everything was fine
<img width="645" alt="Screenshot 2022-11-28 at 08 15 54" src="https://user-images.githubusercontent.com/40179292/204227163-7e4607be-6542-4446-a0f6-0b261f639d7d.png">